collect_app/src/main/java/org/odk/collect/android/utilities/EncryptionUtils.java

```diff
@@ -271,12 +271,9 @@ public static EncryptedFormInformation getEncryptedFormInformation(Uri uri,
         Cursor formCursor = null;
         try {
             if (InstanceColumns.CONTENT_ITEM_TYPE.equals(cr.getType(uri))) {
-                // chain back to the Form record...
-                String[] selectionArgs = null;
-                String selection = null;
-                Cursor instanceCursor = null;
-                try {
-                    instanceCursor = cr.query(uri, null, null, null, null);
+                String[] selectionArgs;
+                String selection = FormsColumns.DELETED_DATE + " IS NULL AND " + FormsColumns.JR_FORM_ID + " =? AND ";
+                try (Cursor instanceCursor = cr.query(uri, null, null, null, null)) {
```

Comment:
I really wanted to get this using the instance/form repositories but it didn't feel worth the effort and risk at this time.